File: program/convert-miles-to-feet/ConvertMilesToFeet.kt
fun main() {
    print("Enter Miles : ")
    println()
    var input = readLine()

    println("$input miles = " + ConvertMilesToFeet(input!!.toDouble()) + " feet")

}

private fun ConvertMilesToFeet(miles: Double): Double {
    var feet = miles * 5280

    return feet
}

File: program/convert-miles-to-feet/convert_miles_to_feet.rs
use std::io;

fn main() {
    println!("Enter miles:");
    let mut input = String::new();
    io::stdin().read_line(&mut input).expect("Failed to read line");
    let miles: f32 = input.trim().parse().expect("Please enter a valid number");

    let feet = miles * 5280.0;

    println!("{} miles = {} feet", miles, feet);
}
